api/src/LinkyLink/Models/Link.cs

```diff
@@ -1,8 +1,11 @@
-﻿namespace LinkyLink.Models
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace LinkyLink.Models
 {
     /// <summary>
     /// Data model class to represent Link in a Link Bundle.
     /// </summary>
+    [ExcludeFromCodeCoverage]
```

Comment:
comment on why we're excluding this

Reply:
These are data model classes and there is no value to test these.